src/components/App/App.test.tsx

```diff
@@ -2,15 +2,23 @@ import { render, screen } from "@testing-library/react"
 import { App } from "./App"
 
 describe("App", () => {
-  it("renders expected text", () => {
-    // Arrange
-    const text = "Learn React"
+  test("renders a banner with header element", () => {
+    render(<App />)
+    expect(screen.getByRole("banner")).toBeInTheDocument()
+  })
 
-    // Act
+  test("renders a logo with alt attribute", () => {
     render(<App />)
-    const received = screen.getByText(text)
+    expect(screen.getByRole("img")).toBeInTheDocument()
```

Comment:
Is this line necessary? In my testing, removing it and failing the test provided the same level of information. Though, I can see that this might follow a progression. What do you think?

Reply:
I like that idea. I wonder if it would make sense to have a dedicated examples directory with various components, utils, etc. with corresponding tests. This directory could be deleted... or maybe it's optionally generated?

My only concern with examples in real application code is that it can feel like a burden to strip it out—for example, the App component will be completely remade in any project that uses this template. The more complex the tests, the harder it is to consider utilizing or morphing existing assertions. Just a thought—not sure I'm right about any of that.